edt/bms/bm_c1a1c.edt

```diff
@@ -4,15 +4,22 @@
 {
 	"chapter" "Unforseen Consequences"
 	"nextmap" "bm_c1a1d"
+	"intro_type" "none"
```

Comment:
Fixed by removing `delay` keyword.

Reply:
This overrides the [base config default](https://github.com/ampreeT/SourceCoop/blob/a8b5839225cb825b944d9a3ad2ad4581ee23e88d/edt/common/base/coop_base_sp_conversion.edt#L7), which is freeze, so it's not the same as not having it.
I dunno about this version, but in the old implementation, this would mean you can progress while intro timer is running.

Reply:
In both the old and new versions of the wait period implementation, `none` will never show a timer and will immediately start if there is no delayed outputs which is what is desired for `bm_c1a1c`, `bm_c1a1d` and `bm_c1a1e`. These maps don't have any cutscenes that the player will miss.

Changed `bm_c1a1b` to use `freeze` instead of `none`.

Reply:
You're right! I've got that confused since the outputhook count check has been added additionally.

I'd still leave it default for consistency (servers can always disable it with `sourcecoop_start_wait_period 0`), but that's just my opinion.

Reply:
Another point to consider:
If players spawn after a checkpoint is activated, they won't carry their equipment over.